src/components/ApplicationDetails/ApplicationDetails.tsx

```diff
@@ -82,14 +85,34 @@ const ApplicationDetails: React.FC<HacbsApplicationDetailsProps> = ({ applicatio
         data-test="application-details-test-id"
         headTitle={appDisplayName}
         breadcrumbs={[
-          { path: '/stonesoup/applications', name: 'Applications' },
+          <Badge key="badge" isRead>
+            WS
+          </Badge>,
+          <span key="badge-divider" className={styles.breadcrumbItemDivider} />,
+          <BreadcrumbItem key="workspace-link" to="#">
```

Comment:
I noticed we're getting react errors since we're using link inside breadcrumbitem
```
Warning: validateDOMNesting(...): <a> cannot appear as a descendant of <a>.
```
I could fix this in the switcher PR.

Reply:
Update: fixed in #394